resources/Materials/TestSuite/stdlib/upgrade/1_36_to_1_37.mtlx

```diff
@@ -1,185 +1,186 @@
-    <input name="in3" type="float" value="0.2500" />
-    <input name="in4" type="float" value="0.7500" />
-  </combine>
-
-  <!-- separate to separate2/separate3/separate4-->
-  <separate name="separate2" type="multioutput">
-    <input name="in" type="color2" value="0.42, 0.77" />
-  </separate>
-  <separate name="separate3" type="multioutput">
-    <input name="in" type="color3" value="0.42, 0.77, 0.93" />
-  </separate>
-  <separate name="separate4" type="multioutput">
-    <input name="in" type="vector4" value="0.42, 0.77, 0.93, 1.0" />
-  </separate>
-
-  <!-- backdrop -->
-  <backdrop name="custom1_backdrop">
-    <parameter name="contains" type="string" value="custom1" />
-    <parameter name="width" type="float" value="20" />
-    <parameter name="height" type="float" value="30" />
-    <parameter name="note" type="string" value="Backdrop for custom1." />
-  </backdrop>
-  
-  <!-- geomattr -->
-  <geominfo name="gi1" geom="/a/g1">
-    <geomattr name="surfid" type="integer" value="15"/>
-  </geominfo>

-  <!-- geomattrvalue -->
-  <geomattrvalue name="srfidval1" type="integer" attrname="surfid"/> 
-  
-</materialx>
+<?xml version="1.0"?>
+<!--
+
+Upgrade path test from 1.36 to 1.37
+
+-->
+<materialx version="1.36">
+
+  <!-- invert matrix to invertmatrix -->
+  <invert name="invert1" type="matrix33">
+    <input name="in" type="matrix33" value="1.0, 0.0, 0.0,  0.0, 1.0, 0.0,  0.0, 0.0, 1.0"/>
+  </invert>
+  <invert name="invert2" type="matrix44">
+    <input name="in" type="matrix44" value="1.0, 0.0, 0.0, 0.0,  0.0, 1.0, 0.0, 0.0,  0.0, 0.0, 1.0, 0.0,  0.0, 0.0, 0.0, 1.0"/>
+  </invert>
+
+  <!-- no change for non-matrix invert variants -->
+  <invert name="invert_no_change_float" type="float">
+    <input name="in" type="float" value="0.5"/>
+  </invert>
+  <invert name="invert_no_change_vec2" type="vector2">
+    <input name="in" type="vector2" value="0.5000, 0.5000" />
+    <parameter name="amount" type="vector2" value="1.0, 0.5000" />
+  </invert>
+  <invert name="invert_no_change_vec3" type="vector3">
+    <input name="in" type="vector3" value="0.5000, 0.5000, 0.0" />
+    <parameter name="amount" type="vector3" value="1.0, 0.5000, 0.0000" />
+  </invert>
+  <invert name="invert_no_change_vec4" type="vector4">
+    <input name="in" type="vector4" value="0.5000, 0.5000, 0.0, 1.0" />
+    <parameter name="amount" type="vector4" value="1.0, 0.5000, 0.0000, 2.0000" />
+  </invert>
+  
+  <!-- rotate to rotate2d and rotate3d -->
+  <rotate name="rotate1" type="vector2">
+    <input name="in" type="vector2" value="0.0, 0.0"/>
+    <input name="amount" type="float" value="1.5708" unittype="angle" unit="radian" />
+  </rotate>
+  <rotate name="rotate2" type="vector3" xpos="5.19248" ypos="5.306">
+    <input name="in" type="vector3" value="0.0, 0.0, 0.0" />
+    <input name="amount" type="float" value="180.0000" unittype="angle" unit="degree" />
+    <parameter name="axis" type="vector3" value="0.0, 1.0000, 1.0000" />
+  </rotate>
```

Comment:
The main updates are to the unit attribute. The rest is some formatting spacing udpate.